src/maheshKJun20/StringExamples/ExtractIdFromString.java

```diff
@@ -0,0 +1,42 @@
+/*
+Extract ItemId from below input
+String input = https://www.facebook.com/technocredits/temId=12345/
+String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+String input = https://www.facebook.com/technocredits/empid=123/
+
+Output : 
+item id is 12345 
+        OR
+no item id present.
+
+*/
+
+package maheshKJun20.StringExamples;
+
+public class ExtractIdFromString {
+	
+	void displayItemId(String str) {
+		System.out.println("Input: " + str);
+		String itemID = "";
+		if (str.contains("itemId=")) {
+			for (int index=str.indexOf("itemId=")+7;index<str.length();index++) {
+				if (String.valueOf(str.charAt(index)).equals("&") || String.valueOf(str.charAt(index)).equals("/")) {
+					break;
+				}
+				itemID += str.charAt(index);
+			}
+			System.out.println("Output: ItemId is - " + itemID);
+		}else
+			System.out.println("Output: No ItemId present in String");
```

Comment:
if (input.contains("itemId=")) {
			String str = input.split("itemId=")[1];
			String output = "";
			for (int index = 0; index < str.length(); index++) {
				if (Character.isDigit(str.charAt(index))) {
					output += str.charAt(index);
				} else {
					break;
				}
			}
			System.out.println("Item id is " + output);
		} else {
			System.out.println("No itemID present.");
		}